src/utils/featheryClient/index.ts

```diff
@@ -474,6 +522,59 @@ export default class FeatheryClient extends IntegrationClient {
     );
   }
 
+  /**
+   * If there is a pending invocation of submitCustom, this method calls it immediately
+   */
+  flushPendingSubmitCustomUpdates(override = true) {
```

Comment:
can we prepend this with an underscore since this is only used by the client internally